ContinuousIntegrationAndDeployment.md

```diff
@@ -0,0 +1,43 @@
+# Continuous Integration/Deployment
+
+## Overview
+
+![CI/CD](ContiniousIntegrationAndDelivery.png)
+
+Continuous Integration/Deployment (CI/CD) is a systematic way to give feedback, iterate, and deploy your changes into your environments. CI/CD can be used to gather telemetry on code changes in a quick and predictable manner.
+
+This solution architecture hooks in Visual Studio Team Services (vsts) into github and uses build phases to gate code checkins into the master branch.
+
+## Flow
+
+### Pull Requests
+
+The feature developer will submit their feature branch and request a `pull request` (PR) against the master branch on `github`. Interested developers will provide feedback and work with the feature developer on implementational changes.
+
+As the feature developer begins making iterations, each additional update to the feature branch will result in a PR build. Some typical steps in this build involve testing, building, and linting the code to ensure no regressions were introduced.
+
+Both parties will continue to iterate on this pull request until both parties have reached an agreement. Interested developers will provide their approval.
```

Comment:
Both parties will continue to iterate on this pull request until ~~both parties~~ they have reached an agreement. Interested developers will provide their approval.